quixstreams/models/serializers/schema_registry.py

```diff
@@ -57,9 +62,25 @@ class SchemaRegistrySerializationConfig(BaseSettings):
         Defines how Schema Registry subject names are constructed. Standard naming
         strategies are defined in the confluent_kafka.schema_registry namespace.
         Defaults to topic_subject_name_strategy.
+    :param skip_known_types: Whether or not to skip known types when resolving
+        schema dependencies. Defaults to False.
+    :param reference_subject_name_strategy: Defines how Schema Registry subject names
+        for schema references are constructed. Defaults to reference_subject_name_strategy.
+    :param use_deprecated_format: Specifies whether the Protobuf serializer should
+        serialize message indexes without zig-zag encoding. This option must be explicitly
+        configured as older and newer Protobuf producers are incompatible.
+        If the consumers of the topic being produced to are using confluent-kafka-python <1.8,
+        then this property must be set to True until all old consumers have been upgraded.
     """
 
     auto_register_schemas: bool = True
     normalize_schemas: bool = False
     use_latest_version: bool = False
     subject_name_strategy: SubjectNameStrategy = topic_subject_name_strategy
+
+    # Protobuf-only atrributes
+    skip_known_types: bool = False
+    reference_subject_name_strategy: ReferenceSubjectNameStrategy = (
+        reference_subject_name_strategy
+    )
+    use_deprecated_format: bool = False
```

Comment:
If these are protobuf-only parameters, should we maybe prefix them with `protobuf_` ?
We could map the names with pydantic easily

Reply:
I am against this, my arguments: 

We mimic the Confluent config 1 to 1, which they did not do on their end. It will confuse our users when they port their configs to Quix.

I will give you an example. Imagine a user with a Protobuf use-case, they don't even know what Avro or JSON Schema is (hypothetically). They'd initialize the config like this:

```python
ser_conf = SchemaRegistrySerializationConfig(
    auto_register_schemas=False,
    normalize_schemas=False,
    subject_name_strategy=some_func,
    protobuf_skip_known_types=True,
    protobuf_reference_subject_name_strategy=some_other_func,
)
```

Some params are prefixed, others not, which is rather inconsistent.